prelude/toolchains/cxx.bzl

```diff
@@ -41,12 +41,12 @@ def _system_cxx_toolchain_impl(ctx):
             linker_info = LinkerInfo(
                 linker = RunInfo(args = ["clang++"]),
                 linker_flags = ["-fuse-ld=lld"],
-                archiver = RunInfo(args = ["ar", "rcs"]),
+                archiver = RunInfo(args = ["ar"]),
                 archiver_type = archiver_type,
                 type = linker_type,
                 link_binaries_locally = True,
                 archive_objects_locally = True,
-                use_archiver_flags = False,
+                use_archiver_flags = True,
```

Comment:
Why is this necessary?

Reply:
See response to https://github.com/facebookincubator/buck2/pull/58/files#r1061631371.